src/js/__tests__/__snapshots__/Components-test.js.snap

```diff
@@ -488,6 +488,8 @@ Object {
       "colors": Object {},
       "size": Object {
```

Comment:
Can we add a test for custom sizes in a theme and for a caller supplied size prop?

Reply:
yeah good idea.